int/codec.ts

```diff
@@ -83,3 +87,25 @@ const _256 = (signed: boolean) => {
 
 export const u256 = _256(false);
 export const i256 = _256(true);
+
+export function int(signed: boolean, size: 8 | 16 | 32): Codec<number>;
```

Comment:
Drastically simpler for visitors that care only if the codec represents a number. I like it!